mkdocs/config/base.py

```diff
@@ -206,12 +206,12 @@ def load_config(config_file=None, **kwargs):
         log.debug(f"Config value: '{key}' = {value!r}")
 
     if len(errors) > 0:
-        raise exceptions.ConfigurationError(
-            f"Aborted with {len(errors)} Configuration Errors!"
+        raise exceptions.Abort(
+            "Aborted with {} Configuration Errors!".format(len(errors))
         )
     elif cfg['strict'] and len(warnings) > 0:
-        raise exceptions.ConfigurationError(
-            f"Aborted with {len(warnings)} Configuration Warnings in 'strict' mode!"
+        raise exceptions.Abort(
```

Comment:
Note: these look like I'm changing from an f-string to `.format`, but actually I just decided to drop this change compared to master.